pkgs/development/python-modules/django-model-utils/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, django
+, freezegun
+, psycopg2
+, pytest-django
+, pytestCheckHook
+, pythonOlder
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "django-model-utils";
+  version = "4.2.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "jazzband";
+    repo = "django-model-utils";
+    rev = version;
+    sha256 = "sha256-TLqvpP/ZaGGFdqnN+UHbhXv1K1YVYTYBkCiWCjYrFh8=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    django
+  ];
+
+  # requires postgres database
+  doCheck = false;
+
+  checkInputs = [
+    freezegun
+    psycopg2
+    pytest-django
+    pytestCheckHook
+  ];
+
```

Comment:
Testing is disabled.

```suggestion

```

Reply:
I kinda want to keep this. Most of the time it is not obvious that pytest-django is required to run the tests.